solr/core/src/java/org/apache/solr/core/ConfigSet.java

```diff
@@ -79,4 +75,14 @@ public NamedList getProperties() {
   public boolean isTrusted() {
     return trusted;
   }
+
+  /**Provide a Schema object on demand
```

Comment:
https://google.github.io/styleguide/javaguide.html#s7-javadoc

Glad to see this supplier though.